modules/python/python-types.c

```diff
@@ -34,15 +34,23 @@
 PyObject *
 py_bytes_from_string(const char *value, gssize len)
 {
+  PyGILState_STATE gstate = PyGILState_Ensure();
+
```

Comment:
I am a bit concerned if this has a runtime overhead for the normal case, e.g. during production, when the gil is already taken. If this is non trivial then I think we should create wrappers instead.

Reply:
This got solved while introducing the `PythonOption` class.